src/mesido/workflows/grow_workflow.py

```diff
@@ -266,6 +266,16 @@ def __state_vector_scaled(self, variable, ensemble_member):
             self.state_vector(canonical, ensemble_member) * self.variable_nominal(canonical) * sign
         )
 
+    def solver_options(self):
+        options = super().solver_options()
+        if options["solver"] == "highs":
+            highs_options = options["highs"]
+            if self.__priority == 1:
+                highs_options["time_limit"] = 100
```

Comment:
just  as a check what happens if the timelimit is reached when there is no feasible solution yet? and what happens if the timelimit is reached and it has a feasible solution but not the mipgap tolerance yet?

Reply:
In both cases it is aborted. Because I only have this small timelimit for priority 1 (e.g. demand matching) I think it should be okay, if one match demand I think this goal should be finished within seconds. (tested on a larger network). I could set the timelimit slightly higher if we'd want.

The only major thing here is to ensure that we agree that if demand is not matched we want the optimisation to be aborted. For users the answer is currently not usefull anyways.

Reply:
agree this is good